### ISSUE TITLE
How do you like this mockup?

### ISSUE THREAD

**Issue by NotGil** [NotGil/CSContest#3]
![mockup](https://cloud.githubusercontent.com/assets/12033870/11901428/340f8368-a571-11e5-8d80-6131c9a32616.PNG)

**Comment by NotGil** [comment]
Below the doted line is what replaces the login screen after succesful login

**Comment by Fyerfox** [comment]
![image](https://cloud.githubusercontent.com/assets/13533940/11915960/4fc4b9fc-a684-11e5-8752-be7a25f43d0b.png)
Made this in HTML, how is it? :o 
Also, the Leaderboard actually works, it takes in JSON and spits it out into that table. Just used monotone colors for simplicity, but can obviously be redone.

**Comment by NotGil** [comment]
Yeah we'll have to add color but it has everything in one place. I think the menu should be left aligned but other than that it looks good

**Comment by Fyerfox** [comment]
Pushed the actual HTML to the repo, and link to live: http://fyerfox.com/cscontest/index-markup.html

**Comment by NotGil** [comment]
![dummy](https://cloud.githubusercontent.com/assets/12033870/11936822/ed535e4c-a7d5-11e5-9f53-4678a6182be6.PNG)
